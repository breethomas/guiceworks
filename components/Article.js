// @flow
import React from 'react'
import withFadeIn from './withFadeIn'

type Props = {
  isHidden: boolean,
}

const Article = ({ isHidden, ...props }: Props) => (
  <article {...props} />
)

export default withFadeIn(Article)

