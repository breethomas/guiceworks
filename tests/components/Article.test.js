import React from 'react'
import renderer from 'react-test-renderer'
import Article from '../../components/Article'

describe('Article', () => {
  let component

  beforeEach(() => {
    component = <Article />
  })

  it('shows the correct snapshot for the Article component tree', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
