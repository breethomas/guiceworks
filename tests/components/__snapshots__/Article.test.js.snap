// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article shows the correct snapshot for the Article component tree 1`] = `
<article
  className="css-1d3ulb6 css-1jyyyen"
/>
`;
